app/views/_mobile-service-instance-row.html

```diff
@@ -37,21 +37,39 @@ <h3>
   </div>
   <div class="list-pf-expansion collapse" ng-if="row.expanded" ng-class="{ in: row.expanded }">
     <div class="list-pf-container">
+
       <div class="expanded-section no-margin">
-        <h4 ng-if="(row.bindings | size) && row.bindingInProgress == false" class="component-label section-label">Details</h4>
-        <ul ng-if="(row.bindings | size) && row.bindingInProgress == false">
-          <li ng-if="row.serviceClass.spec.externalMetadata.documentationUrl | size">
-            <b>Documentation</b>:
-            <a href="{{row.serviceClass.spec.externalMetadata.documentationUrl}}">SDK Setup
-              <i class="fa fa-external-link" aria-hidden="true"></i>
-            </a>
-          </li>
-          <li ng-repeat="annotation in row.annotations">
-            <b>{{annotation.label}}</b>:
-            <a ng-if="annotation.type === 'href'" href="{{annotation.value}}">{{ annotation.value }}</a>
-            <span ng-if="annotation.type === 'string'">{{ annotation.value }}</span>
-          </li>
-        </ul>
+
+        <div class="row" ng-if="(row.bindings | size) && row.bindingInProgress == false">
+          <div class="col-md-6">
+            <h4 class="component-label section-label">Details</h4>
+            <dl class="dl-horizontal left">
+              <dt>Documentation</dt>
+              <dd>
+                <a href="{{row.serviceClass.spec.externalMetadata.documentationUrl}}">SDK Setup
+                  <i class="fa fa-external-link" aria-hidden="true"></i>
+                </a>
+              </dd>
+
+              <dt ng-repeat-start="annotation in row.annotations">{{annotation.label}}</dt>
+              <dd ng-repeat-end>
+                <a ng-if="annotation.type === 'href'" href="{{annotation.value}}">{{ annotation.text || annotation.value }}</a>
+                <span ng-if="annotation.type === 'string'">{{ annotation.value }}</span>
+              </dd>
+            </dl>
+          </div>
+
+          <div class="col-md-6" ng-if="row.extendedAnnotationsType === 'unifiedpush'">
+            <h4 class="component-label section-label">Variants</h4>
```

Comment:
I'm not thrilled with having code specific to a particular service hard-coded into the UI, I imagine we'd have a tough time getting this accepted upstream.

Reply:
@philbrookes that's something I was also concerned. I was asking that to @david-martin during a meeting and he told me that Craig and him discussed about that and this is what we can do initially.

Reply:
@philbrookes  We have specific code for mobile cicd as well